showcase/src/app/demo-ui/pages/collapsible/demo-collapsible-page.component.html

```diff
@@ -18,7 +18,7 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 	</example-viewer>
 
 	<example-viewer
-		[extensions]="['HTML', 'CSS', 'TS']"
+		[extensions]="['HTML', 'SCSS', 'TS']"
```

Comment:
Same here, looking at the SCSS code, I think we can simply remove this file: `showcase/src/assets/examples/collapsible/collapsible-custom.scss` because those styles are not relevant for the example